server/lib/pbench/indexer.py

```diff
@@ -759,10 +759,10 @@ def actions_tracking_closure(cl_actions):
         try:
             resp = resp_payload[_op_type]
         except KeyError as e:
-            assert not ok, "{!r}".format(ok)
-            assert e.args[0] == _op_type, "e.args = {!r}, _op_type = {!r}".format(
-                e.args, _op_type
-            )
+            assert not ok, f"ok = {ok!r}, e = {e!r}"
```

Comment:
So these assertions remain because they have not triggered yet and they are not expected to trigger ever?

Reply:
Yes.